src/template.rs

```diff
@@ -0,0 +1,56 @@
+use handlebars::{handlebars_helper, Handlebars};
+use rand::seq::SliceRandom;
+
+pub fn addhelpers(x: &mut Handlebars) {
+    handlebars_helper!(wrap: |index: usize, s: String| {
+        let mut data = s.clone();
+        if s.len() > index + 1 {
+            data.insert(index, '\n')
+        } else {
+            data.push('\n');
+        }
+        data
+    });
+    handlebars_helper!(wrap_with: |index: usize, ending: String, s: String| {
+        let mut data = s.clone();
+        if s.len() > index + 1 {
+            for ch in ending.chars().rev() {
+                data.insert(index, ch);
+            }
+        } else {
+            for ch in ending.chars() {
+                data.push(ch);
+            }
+        };
+        data
+    });
+    handlebars_helper!(contains: |sub_str: String, content: String | content.contains(&sub_str));
```

Comment:
hmm, should the following be moved to strings. they sound more like string template functions

Reply:
I feel like this function will be used in templates in `if` conditions and it does not return a string but a boolean value.